src/Components/Pages/Profile.js

```diff
@@ -0,0 +1,14 @@
+import RocketProfile from '../RocketProfile';
+import MissionProfile from '../missionProfile';
+
+const Profile = () => (
+  <main className="grid sm:grid-cols-2 grid-cols-1">
+    <div>
+      <MissionProfile />
+    </div>
+    <div>
+      <RocketProfile />
```

Comment:
- [ ] [optional] It would be nice  adding a placeholder message when the "My Missions" or "My Rockets" lists are empty to make the design pretty.